pkg/kv/kvserver/replica_raft.go

```diff
@@ -1302,15 +1309,12 @@ func (r *Replica) handleRaftReadyRaftMuLocked(
 				refreshReason = reasonNewLeaderOrConfigChange
 			}
 		}
-
-		r.mu.raftTracer.MaybeTraceApplied(msgStorageApply.Entries)
-		// Send MsgStorageApply's responses.
-		r.sendRaftMessages(ctx, msgStorageApply.Responses, nil /* blocked */, true /* willDeliverLocal */)
```

Comment:
Were there never any msgStorageApply responses? Or has this functionality moved elsewhere and I've missed it?

Reply:
oh I see, `AckApplied`, nevermind.